collections/Hamilton/MS_Hamilton_49.xml

```diff
@@ -149,33 +198,76 @@
                            <support>paper</support>
                         </supportDesc>
                      </objectDesc>
+                     <handDesc>
+                        <handNote script="cursiva">Gothic cursiva; first written by Henricus Mulhusen of Erfurt (fol. 1r).</handNote>
```

Comment:
add persName role="scr" markup and create authority record
also how does this fit with the above comment about part 1 being written by Indagine? is the folio ref here wrong?

Reply:
It's a note added to the contents list (I think in Indagine's hand) – hopefully clearer now. (Henricus is not in VIAF etc.)

Reply:
Perhaps this might say: "A note by Johannes Hagen on fol. 1r identifies the scribe of fols. 241r-298r and 307r–357r as Henricus Mulhusen of the Erfurt Charterhouse".
I've created a record for him as person_3424

Reply:
Much better – thank you!